File: Examples/DotNET/SDK/VisualBasic/Document/SplitResources/SplitToNewPDFs.vb
Imports Aspose.Cloud

Namespace Document.SplitResources
    Class SplitToNewPDFs
        Public Shared Sub Run()
            Dim inputfile As String = "doc-sample.doc"

            ' Upload input file from local directory to Cloud Storage
            Common.UploadFile(inputfile, String.Empty)

            ' Split all Pages to new PDFs
            Dim wordsSplitResultResponse As WordsSplitResultResponse = Common.WordsService.SplitDocument(inputfile, SplitDocumentFormat.Pdf, 1, 2, Common.FOLDER)

            Console.WriteLine((Convert.ToString(vbLf) & inputfile) + " splitted successfully to new PDFs")
        End Sub
    End Class
End Namespace

File: Examples/DotNET/SDK/CSharp/Document/SplitResources/SplitToNewPDFs.cs
using System;
using Aspose.Cloud;

namespace CSharp.Document.SplitResources
{
    class SplitToNewPDFs
    {
        public static void Run()
        {           
            string inputfile = "doc-sample.doc";

            // Upload input file from local directory to Cloud Storage
            Common.UploadFile(inputfile, string.Empty);

            // Split all Pages to new PDFs
            WordsSplitResultResponse wordsSplitResultResponse = Common.WordsService.SplitDocument(inputfile, SplitDocumentFormat.Pdf, 1, 2, Common.FOLDER);
             
            Console.WriteLine("\n" + inputfile + " splitted successfully to new PDFs");
        }
    }
}
